libs/mve/scene.cc

```diff
@@ -120,19 +96,18 @@ Scene::get_total_mem_usage (void)
 std::size_t
 Scene::get_view_mem_usage (void)
 {
-    std::size_t ret = 0;
-    for (std::size_t i = 0; i < this->views.size(); ++i)
-        if (this->views[i] != nullptr)
-            ret += this->views[i]->get_byte_size();
-    return ret;
+    return std::accumulate(views.begin(), views.end(), std::size_t{0},
```

Comment:
It was not a conscient decision.
I got into the habit of using list initialization, because this form forbids narrowing conversions.